Composer/packages/ui-plugins/virtual-assistant-creation/src/shared/constants.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export enum RouterPaths {
```

Comment:
this can be a simple object, why enum?

Reply:
@a-b-r-o-w-n 
```
{
  "rules": {
    "no-restricted-syntax": [
      "error",
      {
        "selector": "TSEnumDeclaration",
        "message": "Enums are ugh!"
      }
    ]
  }
}
```

Reply:
Enums are fine to me, especially for shared keys. In this case, I think an object is fine though.